lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMPass.cpp

```diff
@@ -372,6 +372,8 @@ class ConvertTritonGPUToLLVM
       auto loadOp = builder.create<triton::LoadOp>(
           insertSliceAsyncOp.getLoc(), tmpTy, insertSliceAsyncOp.getSrc(),
           insertSliceAsyncOp.getMask(), insertSliceAsyncOp.getOther(),
+          // TODO(Chenggang): confirm `boundaryCheck` and `padding`
```

Comment:
what is the meaning of this todo?

Reply:
Currently, we just set them into `nullptr` because the rewrite pass has already eliminated all `boundaryCheck` and `padding` into legacy modes. But while merging the Hopper code (maybe long time later), we have to define `boundaryCheck` and `padding` inside `InsertSliceAsyncOp` and pass the right value here. So I just left a TODO here.